recipes/pyobjcryst/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+export CPATH="${PREFIX}/include:$CPATH"
+export LIBRARY_PATH="${PREFIX}/lib:$LIBRARY_PATH"
+
+MYNCPU=$(( (CPU_COUNT > 4) ? 4 : CPU_COUNT ))
+
+if [ `uname` == Darwin ]; then
+    export DYLD_FALLBACK_LIBRARY_PATH="${PREFIX}/lib"
+fi
```

Comment:
Add `toolchain` as a build requirement and you can remove the lines above (except for `MYNCPU`)